src/main/java/com/ably/kafka/connect/ChannelConfig.java

```diff
@@ -0,0 +1,76 @@
+package com.ably.kafka.connect;
+
+import io.ably.lib.types.AblyException;
+import io.ably.lib.types.ChannelMode;
+import io.ably.lib.types.ChannelOptions;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import static com.ably.kafka.connect.ChannelSinkConnectorConfig.CHANNEL_CONFIG;
+import static com.ably.kafka.connect.ChannelSinkConnectorConfig.CLIENT_CHANNEL_CIPHER_KEY;
+import static com.ably.kafka.connect.ChannelSinkConnectorConfig.CLIENT_CHANNEL_PARAMS;
+
+interface ChannelConfig {
+    String getName();
+
+    ChannelOptions getOptions() throws ChannelSinkConnectorConfig.ConfigException;
+}
+
+class ChannelConfigImpl implements ChannelConfig {
+    private final ChannelSinkConnectorConfig sinkConnectorConfig;
+
+    public ChannelConfigImpl(ChannelSinkConnectorConfig sinkConnectorConfig) {
+        this.sinkConnectorConfig = sinkConnectorConfig;
+    }
+
+    @Override
+    public String getName() {
+        return sinkConnectorConfig.getString(CHANNEL_CONFIG);
+    }
+
+    public ChannelOptions getOptions() throws ChannelSinkConnectorConfig.ConfigException {
+        final Logger logger = LoggerFactory.getLogger(ChannelSinkConnectorConfig.class);
+        ChannelOptions opts;
```

Comment:
There is no need to abbreviate this variable name. In my opinion it makes this code much less readable. I would rename to either `options` or even `channelOptions`.

Reply:
https://github.com/ably/kafka-connect-ably/pull/50/commits/5ab746257b1e1695cf79084ecca2c10f3180053a